macOS setup experience guide: Document order (#32341)
fleetdm/fleet: articles/macos-setup-experience.md

_Available in Fleet Premium_

In Fleet, you can customize the out-of-the-box macOS setup.

> Here's you can configure to happen, and in what order, to your macOS hosts during setup:

Reviewer: Typo here. Would fix myself, but not sure what you are trying to say.

3. Select **Upload** and choose your bootstrap package.

## Software and script

You can configure software installations and a script to be executed during Setup Assistant. This capability allows you to configure your end users' machines during the unboxing experience, speeding up their onboarding and reducing setup time.

If you configure software and/or a script for setup experience, users will see a window like this pop open after their device enrolls in MDM via ADE:

![screen shot of Fleet setup experience window](../website/assets/images/articles/install-software-preview-462x364@2x.png)

This window shows the status of the software installations as well as the script exectution. Once all steps have completed, the window can be closed and Setup Assistant will proceed as usual.

> To replace the Fleet logo with your organization's logo, head to **Settings** > **Organization settings** > **Organization info**, add URLs to your logos in the **Organization avatar URL (for dark backgrounds)** and **Organization avatar URL (for light backgrounds)** fields, and select **Save**. See [configuration documentation](https://fleetdm.com/docs/configuration/yaml-files#org-info) for recommended logo sizes.

Reviewer: ```suggestion
To replace the Fleet logo with your organization's logo:

1. Go to **Settings** > **Organization settings** > **Organization info**
2. Add URLs to your logos in the **Organization avatar URL (for dark backgrounds)** and **Organization avatar URL (for light backgrounds)** fields
3. Press **Save**

> See [configuration documentation](https://fleetdm.com/docs/configuration/yaml-files#org-info) for recommended logo sizes.
```

> The setup experience script always runs after setup experience software is installed. Currently, software that [automatically installs](https://fleetdm.com/guides/automatic-software-install-in-fleet) and scripts that [automatically run](https://fleetdm.com/guides/policy-automation-run-script) are also installed and run during Setup Assistant but won't appear in the window. Automatic software and scripts may run before or after setup the experience software/script. They aren't installed/run in any particular order.

### Install software

To configure software to be installed during setup experience:

> 1. Click on the "Controls" tab in the main navigation bar. Click on "Setup experience", and then on "4. Install software".

Reviewer: ```suggestion
1. Click on the **Controls** tab in the main navigation bar,  then **Setup experience** > **4. Install software**.
```

> 2. Click the "Add software" button. In the modal, select the software that you want to have installed during the setup experience. You can search the list of software by using the search bar in the modal. Click "Save" to save your selection and close the modal.

Reviewer: ```suggestion
2. Click **Add software**, then select or search for the software you want installed during the setup experience.
3. Press **Save** to save your selection.
```

### Run script

To configure a script to run during setup experience:

> 1. Click on the "Controls" tab in the main navigation bar. Click on "Setup experience", and then on "5. Run script".

Reviewer: ```suggestion
1. Click on the **Controls** tab in the main navigation bar, then **Setup experience** > **5. Run script**.
```

> 2. Click "Upload" and select a script (.sh file) from the file picker modal. Once the script is uploaded, you can use the buttons on the script in the web UI to download or delete the script.

Reviewer: ```suggestion
2. Click **Upload** and select a script (.sh file) from the file picker modal. 

> Once the script is uploaded, you can use the buttons on the script in the web UI to download or delete the script.
```
